compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -775,7 +775,7 @@ static SmallVector<int64_t> getDefaultMatmulCacheSizes(linalg::LinalgOp op,
 static LogicalResult setMatmulPeelingRootConfig(
     func::FuncOp entryPointFn, linalg::ContractionOpInterface op,
     ArrayRef<int64_t> distTileSizes, ArrayRef<int64_t> cacheTileSizes,
-    const ScalableTileFlagsListTypeRef inputScalableTileFlags,
+    SmallVector<bool> &inputVecScalableTileFlags,
```

Comment:
Should this be `ArrayRef<bool>` type? Otherwise we could accidentally update the data.